internal/pkg/config/config.go

```diff
@@ -44,11 +44,12 @@ func init() {
 
 // Config contains matchers and analysis scope information.
 type Config struct {
-	Sources    []sourceMatcher
-	Sinks      []funcMatcher
-	Sanitizers []funcMatcher
-	FieldTags  []fieldTagMatcher
-	Exclude    []funcMatcher
+	Sources              []sourceMatcher
+	Sinks                []funcMatcher
+	Sanitizers           []funcMatcher
+	FieldTags            []fieldTagMatcher
+	Exclude              []funcMatcher
+	DontTreatPanicAsSink bool
```

Comment:
Couldn't think of a better name than this. Open to suggestions.

Reply:
Maybe `AllowPanicOnTaintedValues` or something along those lines :thinking:

Reply:
My guidelines are (a) have an intentional zero-value default, and (b) try to speak to the affirmative rather than the negative.

So +1 `AllowPanic*` if we want to _not_ allow that behavior by default.  Otherwise, inverting it to `TreatPanicAsSink` if we want to _not_ treat a panic as a sink by default.  \</two cents\>

Reply:
Agreed. I think "panic is a sink" is the safe default, hence I suggested `Allow*`.